packages/amplify-cli/src/commands/push.ts

```diff
@@ -43,11 +43,11 @@ const syncCurrentCloudBackend = async (context: $TSContext): Promise<void> => {
       securityChangeNotified = await notifyListQuerySecurityChange(context);
     }
 
-    spinner.start(`Fetching updates to backend environment: ${currentEnv} from the cloud.`);
+    printer.info(`Fetching updates to backend environment: ${currentEnv} from the cloud.`);
```

Comment:
spinners tend to crash on windows after a long running deployment, and does not appear to be thread safe; this replaces spinners with printer until we find an alternative